stubs/capturer/capturer.pyi

```diff
@@ -0,0 +1,118 @@
+from _typeshed import FileDescriptorOrPath, SupportsWrite
+from collections.abc import Callable
+from io import BufferedReader
+from multiprocessing import Process
+from multiprocessing.queues import Queue
+from multiprocessing.synchronize import Event
+from signal import Signals
+from types import FrameType, TracebackType
+from typing import IO, Any, NoReturn
+from typing_extensions import Self
+
+__version__: str
+aliases: dict[str, Any]
+DEFAULT_TEXT_ENCODING: str
+GRACEFUL_SHUTDOWN_SIGNAL: Signals
+TERMINATION_DELAY: float
+PARTIAL_DEFAULT: bool
+STDOUT_FD: int
+STDERR_FD: int
+
+def enable_old_api() -> None: ...
+def create_proxy_method(name: str) -> Callable[..., Any]: ...
+
+class MultiProcessHelper:
+    processes: list[Process]
+    def __init__(self) -> None: ...
+    def start_child(self, target: Callable[[Event], Any]) -> None: ...
+    def stop_children(self) -> None: ...
+    def wait_for_children(self) -> None: ...
+    def enable_graceful_shutdown(self) -> None: ...
+    def raise_shutdown_request(self, signum: int, frame: FrameType | None) -> NoReturn: ...
+
+class CaptureOutput(MultiProcessHelper):
+    chunk_size: int
+    encoding: str
+    relay: bool
+    termination_delay: float
+    pseudo_terminals: list[PseudoTerminal]
+    streams: list[tuple[int, Stream]]
+    stdout_stream: Stream
+    stderr_stream: Stream
+    output: PseudoTerminal
+    output_queue: Queue[tuple[Any, bytes]]
+    stdout: PseudoTerminal
+    stderr: PseudoTerminal
+    def __init__(
+        self, merged: bool = ..., encoding: str = ..., termination_delay: float = ..., chunk_size: int = ..., relay: bool = ...
```

Comment:
You can add the runtime values of the defaults; it helps IDEs.
```suggestion
        self, merged: bool = True, encoding: str = ..., termination_delay: float = ..., chunk_size: int = 1024, relay: bool = True
```

Reply:
I have changed it as requested. However, I have some doubts about whether the default value should be retained or not. I also tend to leave functions with their default values. I even want to keep the constants in type annotations so I can replace the remaining ellipses. However, some problems may arise if the author modifies these values. Is there any fundamental difference between retaining a function's default value and retaining a constant value in code?

Reply:
I'm not sure I fully understand your question, but it's true that the defaults may change over time. We have tooling (stubtest) that checks that the defaults are equivalent, so we can update them if necessary.